src/macaron/dependency_analyzer/cyclonedx_mvn.py

```diff
@@ -52,6 +52,12 @@ def get_cmd(self) -> list:
     def collect_dependencies(self, dir_path: str) -> dict[str, DependencyInfo]:
         """Process the dependency JSON files and collect direct dependencies.
 
+        We allow the dependency JSON files to be accepted as long as there is only one JSON file in the target
+        directory. The name of the file is not considered because projects can be configured to produce a custom named
+        SBOM, which cannot be overridden if included at the parent POM level.
+        The presence of multiple JSON files within a target directory differs too greatly from the expectations of the
+        plugin's output. It is for this reason that an error is thrown in such cases.
```

Comment:
```suggestion
        plugin's output. It is for this reason that these cases are treated as error and this method will return an empty dictionary.
```
I think it's better to be explicitly on how the error are handled. We don't throw any error in this method, but only return an empty dictionary.